projects/extension/src/containers/Options.tsx

```diff
@@ -26,6 +26,8 @@ const Options: React.FunctionComponent = () => {
   const [showModal, setShowModal] = useState<boolean>(false)
   const [bg, setBg] = useState<Background | undefined>()
 
+  const [clientError, setClientError] = useState<string>("")
```

Comment:
```suggestion
  const [clientError, setClientError] = useState<string | undefined>(undefined)
```

Why not leave it like it is? `undefined` means "no error". A string, even if empty, means "there's an error".